.github/workflows/agents-publish.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       - run: corepack enable
       - uses: actions/setup-node@v3
         with:
-          node-version: "18"
+          node-version: "20"
```

Comment:
ArrayBuffer resizing is only in Node 20+

Reply:
ok with the upgrade of minimum Node version @coyotte508?

Reply:
yes no problem